.github/workflows/inductor_xpu_e2e_nightly.yml

```diff
@@ -214,18 +230,37 @@ jobs:
     needs: Inductor-XPU-E2E-Nightly-Tests
     steps:
       - name: Report github issue for XPU OPS nightly
-        if: ${{ needs.Inductor-XPU-E2E-Nightly-Tests.result == 'success' }} && ${{ !inputs.suite }}
-        run: |
-          build_url="${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
-          repo="${{ github.repository }}"
-          gh --repo $repo issue comment 304  \
-              --body "XPU OPS Nightly Successful $(date +'%F'), See: $build_url"
-      - name: Report github issue when failure detected
-        if: ${{ needs.Inductor-XPU-E2E-Nightly-Tests.result == 'failure' }} && ${{ !inputs.suite }}
         run: |
+          set -xe
+          if [ "${GITHUB_EVENT_NAME}" == "workflow_dispatch" ];then
+            report_issue_id=426
+            test_type="On-demand"
+          else
+            report_issue_id=432
+            test_type="Nightly"
+          fi
+          if [ "${{ needs.Inductor-XPU-E2E-Nightly-Tests.result }}" == "success" ];then
+            test_status=Success
+          elif [ "${{ needs.Inductor-XPU-E2E-Nightly-Tests.result }}" == "failure" ];then
+            test_status=Failure
+            cc_comment="CC ${{ secrets.NIGHTLY_EMAIL_LIST }}"
+          else
+            test_status=None
+          fi
+          if [ "${GITHUB_EVENT_NAME}" == "workflow_dispatch" ];then
+            cc_comment="CC @${GITHUB_TRIGGERING_ACTOR}"
+            test_scope="Inputs: suite=${{ inputs.suite }}; dt=${{ inputs.dt }}; mode=${{ inputs.mode }}; scenario=${{ inputs.scenario }}; triton=${{ inputs.triton }}; model=${{ inputs.model }}"
```

Comment:
Can we generate a markdown table for inputs parameters and SW/HW info? It will be more clear for readers